build/build.ps1

```diff
@@ -133,7 +133,6 @@ function Build-VS() {
 $all_ok = $True
 
 Build-One '../QsCompiler.sln'
```

Comment:
QIR.sln was removed. Is that on purpose?

Reply:
Oh no! That was an accident. Thanks for the catch, it was just a temporary change to avoid a circular dependency.